### PR TITLE
Updating cli docs for octo.exe

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the command line docs for octo.exe.

Created by TeamCity project 'Automated Documentation Updates::Update CLI Docs', build 191

### CODE REVIEW

File: docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ description: Allows a release to progress to next phase
 The [Octopus CLI](/docs/octopus-rest-api/octopus-cli/index.md) can be used to allow a release to progress to next phase.
 
 ```text
-Allows a release to progress to next phase.
+Allow a release from progressing to next phase.
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md]: @andyinaus I'm struggling to parse this sentence. I think some alternatives are:
Allow a release to progress to the next phase.
Allow a release to move from progressing to the next phase.
Is that something you can change in the CLI tool?

File: docs/octopus-rest-api/octopus-cli/complete.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: complete
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/complete.md]:
```suggestion
title: Complete
```

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 3]
```diff
@@ -11,37 +11,39 @@ hideInThisSection: true
 
 `octo` supports the following commands:
 
-- **[allow-releaseprogression](/docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md)**: Allows a release to progress to next phase.
+- **[allow-releaseprogression](/docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md)**:  Allow a release from progressing to next phase.
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]: Same comment here as above.

File: docs/octopus-rest-api/octopus-cli/index.md [hunk 4]
```diff
@@ -11,37 +11,39 @@ hideInThisSection: true
...
-- **[list-environments](/docs/octopus-rest-api/octopus-cli/list-environments.md)**:  Lists environments.
-- **[list-latestdeployments](/docs/octopus-rest-api/octopus-cli/list-latestdeployments.md)**:  Lists the releases last-deployed in each environment.
+- **[install-autocomplete](/docs/octopus-rest-api/octopus-cli/install-autocomplete.md)**:  Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
+- **[list-deployments](/docs/octopus-rest-api/octopus-cli/list-deployments.md)**:  List a number of deployments by project, environment or by tenant.
+- **[list-environments](/docs/octopus-rest-api/octopus-cli/list-environments.md)**:  List environments.
+- **[list-latestdeployments](/docs/octopus-rest-api/octopus-cli/list-latestdeployments.md)**:  List the releases last-deployed in each environment.
 - **[list-machines](/docs/octopus-rest-api/octopus-cli/list-machines.md)**:  Lists all machines.
 - **[list-projects](/docs/octopus-rest-api/octopus-cli/list-projects.md)**:  Lists all projects.
-- **[list-releases](/docs/octopus-rest-api/octopus-cli/list-releases.md)**:  Lists releases by project.
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/index.md]: The two commands above this should probably be modified to match this pattern.

File: docs/octopus-rest-api/octopus-cli/install-autocomplete.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: install-autocomplete
+description: Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
+---
+
+Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/install-autocomplete.md]:
```suggestion
Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash, and powershell.
```

File: docs/octopus-rest-api/octopus-cli/install-autocomplete.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+**install-autocomplete options**
+
+```text
+Please specify the type of shell to install autocompletion for: --shell=XYZ. Valid values are Pwsh, Zsh, Bash and Powershell.
```
Comment (wordlee) [inline docs/octopus-rest-api/octopus-cli/install-autocomplete.md]:
```suggestion
Please specify the type of shell to install autocompletion for: --shell=XYZ. Valid values are Pwsh, Zsh, Bash, and Powershell.
```

File: docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md (hunk 1 above)
Comment (andyinaus) [reply docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md]: It's coming from the CLI command description, which I just updated previously but having some issues with the release, and I haven't got time to address the release.

File: docs/octopus-rest-api/octopus-cli/index.md (hunk 3 above)
Comment (andyinaus) [reply docs/octopus-rest-api/octopus-cli/index.md]: Same reason as above.

**Comment by andyinaus** [comment]
> Some comments throughout for change requests.

Is Octobob going to pick up the new changes and update this PR?

**Comment by andyinaus** [comment]
Going to close this PR as a new [one](https://github.com/OctopusDeploy/docs/pull/666) has been created

**Pull request closed without merging**